### PR TITLE
Pbi #614

### PR DESCRIPTION
**Pull request by r-kilimov**
Rename lab if it’s “Self” or “Other” to be “RGL”.
Also renamed events name on [wiki page](https://github.com/SarineTechnologies/sarine.viewer.customer.configurations/wiki/Client-Extensions).

### CODE REVIEW

File: app/js/app.dashboard.js [hunk 1]
```diff
@@ -126,9 +134,9 @@
         var overrideTextsPromise = $.getJSON(urls.overrideTexts);
 
         var extensionEvents = {
-            beforeDataLoaded: 'extensions_beforeDataLoaded',
-            beforeMainLoaded: 'extensions_beforeMainLoaded',
-            afterMainLoaded: 'extensions_afterMainLoaded'
+            beforeDataLoaded: 'extensions_before_resources_Loaded',
```
Comment (adica) [inline app/js/app.dashboard.js]: @r-kilimov 
make sure Ido and ben know about this signature change - they need to fix all js extensions files

**Comment by adica** [comment]
@r-kilimov 
it seems that this PR is also about new "any-only" experiences?

File: app/js/app.dashboard.js [hunk 2]
```diff
@@ -126,9 +134,9 @@
         var overrideTextsPromise = $.getJSON(urls.overrideTexts);
 
         var extensionEvents = {
-            beforeDataLoaded: 'extensions_beforeDataLoaded',
```
Comment (aharonam) [inline app/js/app.dashboard.js]: please change the keys also to a meaningfull. for example: beforeDataLoaded -> beforeResourceLoaded.
please expose 'extensionEvents'  outside so the extension can register to the events by its keys not their values.

**Comment by aharonam** [comment]
please see comments in the commits section

**Comment by r-kilimov** [comment]
Implemented changes as Aharon suggested.

**Pull request closed without merging**